NtupleProducer/python/runJetNTupler.py

```diff
@@ -73,10 +75,14 @@
         TKV = cms.InputTag("ak4L1TKV"),
         PF = cms.InputTag("ak4L1PF"),
         Puppi = cms.InputTag("ak4L1Puppi"),
-        L1Calo = cms.InputTag("ak4L1ICalo"),
-        L1TK = cms.InputTag("ak4L1ITK"),
-        L1PF = cms.InputTag("ak4L1IPF"),
-        L1Puppi = cms.InputTag("ak4L1IPuppi"),
+        #L1Calo = cms.InputTag("ak4L1ICalo"),
+        #L1TK = cms.InputTag("ak4L1ITK"),
+        #L1PF = cms.InputTag("ak4L1IPF"),
+        #L1Puppi = cms.InputTag("ak4L1IPuppi"),
+        L1Calo = cms.InputTag("ak4L1Calo"),
+        L1TK = cms.InputTag("ak4L1TK"),
+        L1PF = cms.InputTag("ak4L1PF"),
+        L1Puppi = cms.InputTag("ak4L1Puppi"),
```

Comment:
Now `L1Calo` is identical to `Calo`, and the same for all others.
I think it makes sense then to just keep one set (either the ones with `L1` in front or the others)